cli/src/wallet_cli/mod.rs

```diff
@@ -177,10 +179,11 @@ pub enum WalletCommand {
     },
     /// Request funds from the faucet.
     Faucet {
-        /// Address the faucet sends the funds to, defaults to the wallet address.
+        /// Address the faucet sends the funds to. If not provided, the command defaults to the wallet address.
         address: Option<Bech32Address>,
-        /// URL of the faucet, default to <http://localhost:8088/api/enqueue>.
-        url: Option<String>,
+        /// URL of the faucet.
+        #[arg(short, long, value_name = "URL", env = "FAUCET_URL", default_value = DEFAULT_FAUCET_URL)]
```

Comment:
Is the default visible in the cli? If not I would put it in the comment back so one sees how the format is (with `/api/enqueue`)

Reply:
yes, it's visible